src/components/Events.module.scss

```diff
@@ -15,8 +17,22 @@
   width: 100%;
   gap: 30px;
   justify-content: space-between;
+  max-width: 100%;
 }
 
 .sport {
   font-weight: bold;
 }
+
+@media only screen and (max-width: $screen-md-min) {
+  .event {
+    margin-left: auto;
+    margin-right: auto;
+    height: 200px;
+    width: 343px;
```

Comment:
For our purpose, this will work for us. In real apps, making the card a fixed size or, for that matter, making anything fixed sizing usually won't work well.